### PR TITLE
chore: [CHK-3500] add event hub connection string to key vault

### PR DESCRIPTION
**Pull request by CianoDanilo**
### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

**Review by EmanueleBVtech — APPROVED** [review]
perfect

File: src/domains/observability/.terraform.lock.hcl [hunk 1]
```diff
@@ -1,6 +1,25 @@
 # This file is maintained automatically by "terraform init".
 # Manual edits may be lost in future updates.
 
+provider "registry.terraform.io/azure/azapi" {
+  version = "2.0.1"
+  hashes = [
+    "h1:NybiPNJ4MsKsTtRUnxV5vrhkbPkK+puqgfa0j0Q5nlU=",
```
Comment (infantesimone) [inline src/domains/observability/.terraform.lock.hcl:7]: required?

File: src/domains/pay-wallet-common/02_security.tf [hunk 2]
```diff
@@ -54,6 +54,13 @@ data "azuread_service_principal" "iac_principal" {
   display_name = "pagopaspa-pagoPA-iac-${data.azurerm_subscription.current.subscription_id}"
 }
 
+data "azurerm_eventhub_authorization_rule" "sender_evt_tx_event_hub_connection_string" {
+  name                = "payment-wallet-evt-tx"
+  namespace_name      = "pagopa-${var.env_short}-itn-observ-evh"
```
Comment (infantesimone) [inline src/domains/pay-wallet-common/02_security.tf:59]:
```suggestion
  namespace_name      = "${local.product}-observ-evh"
```

File: src/domains/pay-wallet-common/02_security.tf [hunk 3]
```diff
@@ -54,6 +54,13 @@ data "azuread_service_principal" "iac_principal" {
...
+  eventhub_name       = "payment-wallet-ingestion-dl"
+  resource_group_name = "pagopa-${var.env_short}-itn-observ-evh-rg"
```
Comment (infantesimone) [inline src/domains/pay-wallet-common/02_security.tf:61]:
```suggestion
  resource_group_name = "${local.product}-itn-observ-evh-rg"
```

**Review by infantesimone — CHANGES_REQUESTED** [review]

File: src/domains/observability/.terraform.lock.hcl (hunk 1 above)
Comment (CianoDanilo) [reply src/domains/observability/.terraform.lock.hcl:7]: I don't know if it is required, it's a change made by an apply/plan

File: src/domains/pay-wallet-common/02_security.tf [hunk 4]
```diff
@@ -54,6 +54,13 @@ data "azuread_service_principal" "iac_principal" {
   display_name = "pagopaspa-pagoPA-iac-${data.azurerm_subscription.current.subscription_id}"
 }
 
+data "azurerm_eventhub_authorization_rule" "sender_evt_tx_event_hub_connection_string" {
+  name                = "payment-wallet-evt-tx"
+  namespace_name      = "${local.product}-observ-evh"
+  eventhub_name       = "payment-wallet-ingestion-dl"
+  resource_group_name = "${local.product}-itn-observ-evh-rg"
```
Comment (infantesimone) [inline src/domains/pay-wallet-common/02_security.tf:61]:
```suggestion
  resource_group_name = "${local.product}-observ-evh-rg"
```

**Comment by infantesimone** [comment]
please add jira task ref in PR name

File: src/domains/observability/.terraform.lock.hcl (hunk 1 above)
Comment (infantesimone) [reply src/domains/observability/.terraform.lock.hcl:7]: revert this change, it is a file from a different domain  not related to the PR (perhaps you need to review the terraform configurations or your local configurations)

Comment (CianoDanilo) [reply src/domains/observability/.terraform.lock.hcl]: Done in [f76c1ae](https://github.com/pagopa/pagopa-infra/pull/2504/commits/f76c1ae1639308e19e1f4bdd9ae1fad39cb4bf03)

**Review by infantesimone — APPROVED** [review]

**Comment by infantesimone** [comment]
please tun pre-commit in order to update docs for pay-wallet domain

**Pull request merged**